health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -173,25 +153,25 @@ const Hypothesis = ({ category, assumptions:initialAssumptions, customProps, onS
                       variation="secondary"
-                      isDisabled={assumptions.length === 3}
+                      isDisabled={isAddNewDisabled}
                       
                     />
                     {showPopUP && <PopUp
                                     className={"popUpClass"}
                                     type={"default"}
-                                    heading={t("Are you sure you want to Delete?")}
+                                    heading={t("CONFIRM_TO_DELETE")}
                                     equalWidthButtons={true}
                                       children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)